packages/react-form-state/src/validators.ts

```diff
@@ -73,11 +73,26 @@ export function validateList<Input extends object, Fields>(
   };
 }
 
+export function validateWithFields<Input, Fields>(
+  matcher: Matcher<Input, Fields>,
+  errorContent: ErrorContent,
+) {
+  return validate(matcher, errorContent) as (
```

Comment:
is there any benefit in writing a test to cover this wrapper? it's just messing with typing information so it's probably not worth while. 

Is there a better way to wrap `validate()`? i wanted to just use typing information, but my earlier naive approach doesn't work

Reply:
eh I'd not bother.